translations/fr.json

```diff
@@ -29,9 +29,16 @@
 
 	"FEELS": "Ressenti {DEGREE}",
 	"PRECIP_POP": "Probabilité de précipitations",
+	"PRECIP_AMOUNT": "Precipitation amount",
```

Comment:
Is that french? ;-) If not, please remove, sicne the english tranlation will be used as a fallback anyway.